shimmingtoolbox/cli/b0map.py

```diff
@@ -14,14 +15,15 @@
 @click.argument("output", type=click.Path(exists=False))
 def main(verbose, coilshims, pulseseq, input, output):
     """
-    Compute b0 fieldmap OUTPUT from a folder of dicom files INPUT.
+    Compute b0 fieldmap ``OUTPUT`` from a folder of dicom files ``INPUT/``.
     """
     if verbose:
         logging.getLogger().setLevel(logging.INFO)
-    logging.info(f'{coilshims}, {pulseseq}, {input}, {output}')
-    map, coilshims, pulseseq = b0map(input, coilshims=coilshims, pulseseq=pulseseq)
-    with open(output, 'w') as out:
-        out.write(str(map)) # definitely not the right file format
+    logging.info(f"{coilshims}, {pulseseq}, {input}, {output}")
+    fieldmap, coilshims, pulseseq = b0map(input, coilshims=coilshims, pulseseq=pulseseq)
+    with open(output, "w") as out:
+        out.write(str(fieldmap))  # definitely not the right file format
+
 
-if __name__ == '__main__':
+if __name__ == "__main__":
```

Comment:
that goes against [our conventions](https://github.com/neuropoly/spinalcordtoolbox/wiki/Single-or-double-quotes) (which,  i agree are not easily found-- because of the redirection to another wiki).